src/openvic-dataloader/File.hpp

```diff
@@ -104,6 +106,7 @@ namespace ovdl {
 
 	protected:
 		const char* _path;
+		std::size_t _buffer_size;
```

Comment:
Minor warning, with MSVC this won't necessarily be default zero-initialised. In practice the `BasicFile` constructors below initialise it, but it might be worth zero-initialising it in `File`'s own constructor (or adding it as an argument to `File`'s constructor, rather than assigning it manually in the `BasicFile` constructors).

Also `_buffer` could be initialised via a `File` constructor arg too, although there's no default initialisation issue there.